src/modules/wasi/WspOneModule.v3

```diff
@@ -329,6 +329,7 @@ def OOB = HostResult.Throw(Trap.new(TrapReason.MEM_OUT_OF_BOUNDS, null, null));
 // Enforces ordering on installation of platform-specific modules.
 component WspOneRegistration {
 	def X = ModuleRegistry.add(WspOneModule.new(), true); // register platform-independent first
+	def M = ModuleRegistry.add(MemoryModule.new("env"), true); // Only register `env::memory` when WASI is enabled.
```

Comment:
Since this "env.memory" is actually part of WASI, what about refactoring so that the `WspOneModule` supplies this import? (I.e. no separate `MemoryModule`)?

Reply:
Yeah, it seems like WASI is a special kind of thing that shouldn't just be keyed by the name of the imported module. I think it makes sense to make it an `ImportProcessor` that runs over imports, controlled by a separate flag.

Reply:
Ah I see. This sounds good to me. Will work on this.